index.html

```diff
@@ -1,138 +1,150 @@
-  <p> Только полезная информация и никакого спама,<br>честное бойскаутское!</p>
-  <form class="mailing-form" action="" method="post">
-  <label class="visually-hidden" for="mailing-email">Email</label>
-  <input class="field" placeholder="email@exemple.com" type="email" name="mailing-email" id="mailing-email" required>
-  <button class="button mailing-button" type="submit">
-    <span class="visually-hidden">Подписаться</span>
-  </button>
-</form>
-</section>
-</main>
-<footer class="page-footer"
-<section class="footer-social">
-  <h2 class="visually-hidden">Давайте дружить</h2>
-  <ul class="footer-social-list">
-    <li class="footer-social-item">
-      <a class="button" href="#">
-        <span class="visually-hidden">Твиттер</span>
-      </a>
-    </li>
-    <li class="footer-social-item">
-      <a class="button" href="#">
-        <span class="visually-hidden">Фейсбук</span>
-      </a>
-    </li>
-    <li class="footer-social-item">
-      <a class="button" href="#">
-        <span class="visually-hidden">Ютьюб</span>
-      </a>
-    </li>
-  </ul>
-<a class="contacts-phone" href="tel:+78128121212">+7 (812) 812-12-12</a>
-<a class="htmlacademy-link" href="https://htmlacademy.ru/">.html academy</a>
-</section>
-</footer>
+      <section class="search">
+        <h2 class="search-tiile">Заинтересовались?</h2>
+        <p>Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+        <a class="search-link" href="">Поиск гостиницы в Седоне</a>
+      </section>
+      <section class="mailing">
+        <h2 class="mailing-tiile">Подпишитесь на рассылку</h2>
+        <p> Только полезная информация и никакого спама,честное бойскаутское!</p>
+        <form class="mailing-form" action="" method="post">
+          <label class="visually-hidden" for="mailing-email">Email</label>
+          <input class="field" placeholder="email@exemple.com" type="email" name="mailing-email" id="mailing-email"
+            required>
+          <button class="button mailing-button" type="submit" value="Подписаться"></button>
+        </form>
+      </section>
+  </main>
+  <footer class="page-footer">
+    <section class="footer-social">
+      <h2 class="visually-hidden">Давайте дружить</h2>
+      <ul class="footer-social-list">
+        <li class="footer-social-item">
+          <a href="#">
+            <span class="visually-hidden">Твиттер</span>
+          </a>
+        </li>
+        <li class="footer-social-item">
+          <a href="#">
+            <span class="visually-hidden">Фейсбук</span>
+          </a>
+        </li>
+        <li class="footer-social-item">
+          <a href="#">
+            <span class="visually-hidden">Ютьюб</span>
+          </a>
+        </li>
+      </ul>
+      <a class="page-footer-contacts" href="tel:+78128121212">+7 (812) 812-12-12</a>
+      <a class="page-footer-htmlacademy" href="https://htmlacademy.ru/">.html academy</a>
```

Comment:
Лучше настолько к контенту не привязываться, а сделать класс более абстрактным (мало ли что там изменится). Например, так:
```suggestion
      <a class="page-footer-developer" href="https://htmlacademy.ru/">.html academy</a>
```

Reply:
Хорошо бы еще и доступность "прокачать" =)
```suggestion
      <a class="page-footer-htmlacademy" href="https://htmlacademy.ru/">
        <span class="visually-hidden">Разработчик: </span>
        .html academy
      </a>
```

Reply:
Ага, добавила)